miasm2/jitter/jitcore_python.py

```diff
@@ -120,23 +121,25 @@ def myfunc(cpu):
                 # Manage resulting address
                 if isinstance(ad, m2_expr.ExprInt):
                     return ad.arg.arg
-                elif isinstance(ad, m2_expr.ExprId):
-                    cur_label = ad.name
+                elif isinstance(ad, m2_expr.ExprLoc):
+                    cur_loc_key = ad.loc_key
+                    if cur_loc_key not in known_loc_keys:
+                        return cur_loc_key
```

Comment:
In the previous version, nothing was returned in this case (but I don't think that function ret is used somewhere). A simple `ret` is then likely enough

Reply:
done